src/anyon/anyon.jl

```diff
@@ -660,13 +776,14 @@ end
 """
     run_job(qpu::AnyonYukonQPU, circuit::QuantumCircuit, shot_count::Integer)
 
-Run a circuit computation on a `QPU` service, repeatedly for the specified
-number of repetitions (shot_count).
-Returns the histogram of the completed circuit calculations, along with the 
-simulation's execution time (in milliseconds) or an error
-message.
-If the circuit received in invalid - for instance, it is missing a `Readout` -
-it is not sent to the host, and an error is throw.
+Submit a circuit to a `QPU` service for execution. The number of circuit executions is
```

Comment:
Submit a circuit to a `QPU` service for execution, **avoiding the standard transpilation performed in transpile_and_run_job()[@ref]**